src/LinQuadOptInterface.jl

```diff
@@ -205,6 +205,8 @@ function Base.show(io::IO, model::LinQuadOptimizer)
     return
 end
 
+MOIU.supports_default_copy_to(model::LinQuadOptimizer, copy_names::Bool) = true
```

Comment:
All the LQOI models support names ?

Reply:
It's better if they all have to explicitetly implement it and LQOI uses `automatic_copy_to` so that there is a nice error message in case names are not supported

Reply:
I'm confused (along with https://github.com/JuliaOpt/MathOptInterface.jl/pull/603/files#r241808786) as to what should be implemented an by whom.

Are you saying that I should implement:
- LQOI
```julia
function MOI.copy_to(dest::LinQuadOptimizer, src::MOI.ModelLike; kws...)
    return MOI.Utilities.automatic_copy_to(dest, src; kws...)
end
```
- Gurobi
```julia
MOIU.supports_default_copy_to(::Gurobi.Optimizer, copy_names::Bool) = true
```

Reply:
> Are you saying that I should implement

Exactly

Reply:
Okay so two questions:
 1. why didn't I have to implement `MOI.copy_to`?
 2. what is the different error message? This is a developer implementing the wrapper message or a user using the wrapper message?

Reply:
1. You are implementing it:
https://github.com/JuliaOpt/LinQuadOptInterface.jl/blob/4e1372226290230959a999fe45998cdf8a1e1eda/src/copy.jl#L2-L11
By the way I don't understand why there is two methods
2. User of JuMP in direct mode:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/c8e36824c4e77aa5993b7c706237074dc87f880c/src/Utilities/copy.jl#L19-L20